W-16770201-rtf-persistence-gateway-updates-kt (#1412)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/persistence-gateway.adoc

To migrate the persistent data from one cluster to another, backup these two tables entirely from the source cluster and recreate them on the target one database. Do not to have applications deployed during this process on both clusters to prevent unexpected results.

> Note that if you are using a version of the Runtime Fabric agent higher than 2.6.22, the `id` parameter in the `CREATE TABLE IF NOT EXISTS items` statement no longer applies. 

Reviewer: What if we phrase this the other way around? The schema with the modifications done in line 171 no longer contains the id definition.

Should the note state that for older versions than 2.6.22, an `id` column will be there for the items table?

Reviewer 2: +1

> Note that if you are using a version of the Runtime Fabric agent higher than 2.6.22, the `id` parameter in the `CREATE TABLE IF NOT EXISTS items` statement no longer applies. For previous versions of the Runtime Fabric agent 2.6.22, the `id` parameter is available in the items table. 

Reviewer 3: ```suggestion
If you're using a version of the Runtime Fabric agent higher than 2.6.22, the `id` parameter in the `CREATE TABLE IF NOT EXISTS items` statement no longer applies. For previous versions of the Runtime Fabric agent 2.6.22, the `id` parameter is available in the items table. 
```

Reviewer 3: If this note is too important, create a note:
[NOTE]
If you're using a version of the Runtime Fabric agent higher than 2.6.22, the `id` parameter in the `CREATE TABLE IF NOT EXISTS items` statement no longer applies. For previous versions of the Runtime Fabric agent 2.6.22, the `id` parameter is available in the items table.
